demos/appyx-navigation/android/src/main/kotlin/com/bumble/appyx/navigation/node/viewModel/MyViewModel.kt

```diff
@@ -0,0 +1,25 @@
+package com.bumble.appyx.navigation.node.viewModel
+
+import com.bumble.appyx.utils.viewmodel.integration.IntegrationPointViewModel
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.getAndUpdate
+import androidx.lifecycle.viewModelScope
+import kotlinx.coroutines.launch
+
+class MyViewModel : IntegrationPointViewModel() {
```

Comment:
I think it should extend a ViewModel instead

Reply:
let's create a proper example: an Activity which uses ViewModelNodeActivity which hosts a ViewModelNode with ViewModel inside?

Reply:
Probably done :)